common/src/main/java/org/opensearch/ml/common/transport/model/MLUpdateModelInput.java

```diff
@@ -32,11 +33,12 @@ public class MLUpdateModelInput implements ToXContentObject, Writeable {
     public static final String MODEL_VERSION_FIELD = "model_version"; // passively set when register model to a new model group
     public static final String MODEL_NAME_FIELD = "name"; // optional
     public static final String MODEL_GROUP_ID_FIELD = "model_group_id"; // optional
+    public static final String IS_ENABLED_FIELD = "is_enabled"; // optional
+    public static final String MODEL_RATE_LIMITER_CONFIG_FIELD = "model_rate_limiter_config"; // optional
     public static final String MODEL_CONFIG_FIELD = "model_config"; // optional
-    public static final String CONNECTOR_FIELD = "connector"; // optional
+    public static final String UPDATED_CONNECTOR_FIELD = "updated_connector"; // passively set when updating the internal connector
```

Comment:
When we update model description, we don't need a new filed `updated_description`. Why do we need this new filed when update connector?

Reply:
Discussed offline with Sicheng. This filed is not user's input. It's constructed by code with the original connector and user's input new connector content (the `CONNECTOR_FIELD`).

Reply:
Generally we use input class only for user's input data. This implementation is different. Suggest add more comments so others know user can't input which fields.

Reply:
If you have more time, suggest add another class for internal usage. Keep this class only for user's input

Reply:
Yeah agree. We can moved out from parser first, but I will investigate another way to handle it.